.github/workflows/main_branch_check.yml

```diff
@@ -63,7 +63,7 @@ jobs:
           username: ${{ env.BROWSERSTACK_USERNAME }}
           access-key: ${{ env.BROWSERSTACK_ACCESS_KEY }}
 
-  scan:
+  scan-sonarcloud:
```

Comment:
Can we use the same name for v7 as well ? Just for consistency.

Reply:
I have update it in v7 PR to be the same for consistency, please review it again.